src/app/Command.cpp

```diff
@@ -31,7 +31,7 @@
 namespace chip {
 namespace app {
 
-CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr)
+CHIP_ERROR Command::Init(Messaging::ExchangeManager * apExchangeMgr, InteractionModelDelegate * apDelegate)
```

Comment:
Can we add another function for setting InteractionModelDelegate?

Reply:
currently we expect there is only one delegate passed from consumer sdk, therefore we don't expect to add additional setDelegate function

Reply:
https://github.com/project-chip/connectedhomeip/issues/5332, we can discuss the necessity of multiple delegate in this ticket, thanks